queries.sql

```diff
@@ -12,4 +12,55 @@ SELECT * FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name != 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;   // const user = userData ? JSON.parse(userData) : null
+
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(100);
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT before_weight_update;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO before_weight_update;
+UPDATE animals SET weight_kg = ABS(weight_kg) WHERE weight_kg < 0;
```

Comment:
👏 You've made a comprehensive analysis of the statement! 👏 However, there's a nuanced difference between your query and the corrected statement. Let's delve into it:

- [ ] **Preserving Negative Sign in Weights:** Your query utilizes the `ABS()` function, which takes the absolute value of the weight, effectively removing the negative sign. However, the project requirement requested to use multiplication by -1, aligning with the project requirement to change the sign to positive using this method.
- [ ] Additionally if necessary please consider updating the Screenshots in the PR Description.

By understanding this distinction, you can choose the approach that best aligns with the specific project requirement. Keep up the meticulous review! 🚀🔍